src/name_bazaar/ui/pages/user_purchases_page.cljs

```diff
@@ -30,41 +31,45 @@
                                      :order-by-dirs [order-by-dir]}])))}]))))
 
 (defn user-purchases []
-  (let [search-results (subscribe [:offerings/user-purchases])]
+  (let [search-results (subscribe [:offerings/user-purchases])
+        route-params (subscribe [:district0x/route-params])]
     (fn [{:keys [:title :no-items-text]}]
       (let [{:keys [:items :loading? :params :total-count]} @search-results]
-        [app-layout
-         [ui/Segment
-          [ui/Grid
-           {:padded true
-            :class :no-inner-horizontal-padding}
-           [ui/GridColumn
-            {:width 16
-             :class :join-upper}
-            [:h1.ui.header title]]
-           [ui/GridColumn
-            {:computer 6
-             :tablet 8
-             :mobile 16
-             :floated "right"}
-            [user-purchases-order-by-select]]]
-          [offering-infinite-list
-           {:class "primary"
-            :total-count total-count
-            :offset (:offset params)
-            :loading? loading?
-            :no-items-text no-items-text
-            :header-props {:show-time-ago? true
-                           :show-sold-for? true}
-            :on-next-load (fn [offset limit]
-                            (dispatch [:offerings.user-purchases/set-params-and-search
-                                       {:offset offset :limit limit} {:append? true}]))}
-           (doall
+        [:div.wrapper
+         [name-bazaar-meta-tags {:title title :description (if-let [address (:user/address @route-params)]
```

Comment:
:title "NameBazaar - [title]"
:description "See all ENS name purchases of [address]"